packages/bot-skeleton/src/scratch/utils/index.js

```diff
@@ -7,6 +7,53 @@ import { removeLimitedBlocks } from '../../utils/workspace';
 import { saveWorkspaceToRecent } from '../../utils/local-storage';
 import DBotStore from '../dbot-store';
 import { log_types } from '../../constants/messages';
+import { error_message_map } from '../../utils/error-config';
+
+export const matchTranslateAttribute = translateString => {
+    const match = translateString.match(/translate\((-?\d*\.?\d+),(-?\d*\.?\d+)\)/);
+
+    if (match) {
+        const x = parseFloat(match[1]);
+        const y = parseFloat(match[2]);
```

Comment:
@rupato-deriv can we add a check for match, if it's length is 2? Otherwise if the length is 1 it may throw an error for y.

Reply:
done ✅